website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -448,6 +450,294 @@ $ consul-k8s proxy read backend-658b679b45-d5xlb -o raw
 }
 ```
 
+### `proxy log`
+
+The `proxy log` command allows you to inspect and modify the logging configuration of Envoy proxies running on a given Pod.
+
+```shell-session
+$ consul-k8s proxy log <PODNAME> <OPTIONS>
+```
+
+The command takes a required value, `<PODNAME>`. This should be the full name
+of a Kubernetes Pod. If a Pod is running more than one Envoy proxy managed by
+Consul, as in the [Multiport configuration](/consul/docs/k8s/connect#kubernetes-pods-with-multiple-ports),
+configuration for all proxies in the Pod will be displayed.
+
+The following options are available.
+
+| Flag                               | Description                                                                                                                                                                                                                                                               | Default                                                                                                                |
+| ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| <nobr>`-namespace`, `-n`</nobr>    | `String` The namespace where the target Pod can be found.                                                                                                                                                                                                                 | Current [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) namespace. |
+| <nobr>`-update-level`, `-u`</nobr> | `String` Update the level for the logger. Can be either `-update-level <LEVEL>` to change all loggers to a specific level, or a comma delineated list of loggers with level can be passed like `-update-level grpc:warning,http:info` to only modify specific loggers.    | none                                                                                                                   |
```

Comment:
> Is "update level" a clear or standard type of logger?

I'm not entirely sure what you're asking, `update-level` is the option here that is used to modify the log level, there is no default here because the user is specifying a level to make a change to (so for example "I want to set the grpc logger to warning" or "I want to change all loggers to debug"). The `reset` command has a default because that is what it changes all log levels